### PR TITLE
Engineering/deploy version

### PR DESCRIPTION
**Pull request by ydrisR**
1. Add a route /version to publish the current version
2. Remove the deployment step as it now done from another pipeline

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ It is based on Tidepool Blip 1.27.
 - YLP-1767 Create package to migrate viz components
 - YLP-1797 Make it possible to see what role the users are logged in with matomo
 - YLP-1834 Migrate time in range stats
+- Add a route /version to display blip version
```
Comment (TimCanu) [inline CHANGELOG.md:27]: Shouldn't there be the JIRA number?

File: cloudfront-dist/deployment/preview.env [hunk 2]
```diff
@@ -1,13 +1,13 @@
 AWS_DEFAULT_REGION=eu-west-1
 STACK_PREFIX_NAME=preview
-STACK_VERSION=2.0.0
+STACK_VERSION=3.0.1
```
Comment (yanngit) [inline cloudfront-dist/deployment/preview.env:3]: it's unused right ? Can't we remove it ?

File: CHANGELOG.md (hunk 1 above)
Comment (ydrisR) [reply CHANGELOG.md:27]: Not needed for engineering

File: cloudfront-dist/deployment/preview.env (hunk 2 above)
Comment (ydrisR) [reply cloudfront-dist/deployment/preview.env]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (TimCanu) [reply CHANGELOG.md:27]: I didn't know that...

Comment (TimCanu) [reply CHANGELOG.md:27]: So much time wasted

Comment (TimCanu) [reply CHANGELOG.md:27]: Are you sure?

**Comment by TimCanu** [comment]
I imagine that this PR is blocked by: https://github.com/mdblp/yourloops/pull/773 ?

**Review by TimCanu — APPROVED** [review]
As discussed, ok for me but make sure that you are not going to break everything by merging this if you are not in sync with yourloops (yourloops build being broken, and BETA should be published before merge otherwise you are taking another risk that can break stuff)

**Pull request merged**